xml/slurm.xml

```diff
@@ -52,13 +52,12 @@
   <sect2 xml:id="sec-slurm-minimal">
    <title>Minimal installation</title>
    <important>
-    <title>Make sure of consistent UIDs and GIDs for Slurm's accounts</title>
+    <title>Make sure of consistent UIDs and GIDs for &slurm;'s accounts</title>
     <para>
-     For security reasons, Slurm does not run as the user
-     <systemitem
-      class="username">root</systemitem> but under its own
-     user. It is important that the user
-     <systemitem class="username">slurm</systemitem> has the same UID/GID
+     For security reasons, &slurm; does not run as the user
+     <systemitem class="username">root</systemitem> but under its own
+     user. It is important that the user <systemitem
+      class="username">slurm</systemitem> has the same UID/GID
```

Comment:
Afaict, this does not change content, so I would avoid this spurious reformatting:

```suggestion
     For security reasons, Slurm does not run as the user
     <systemitem
      class="username">root</systemitem> but under its own
     user. It is important that the user
     <systemitem class="username">slurm</systemitem> has the same UID/GID
```